tasks/sep/base-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,341 @@
+[factory-etherscan]: https://sepolia.etherscan.io/address/0xb1efB9650aD6d0CC1ed3Ac4a0B7f1D5732696D37
+[output-oracle-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L6
+[output-oracle-etherscan]: https://sepolia.etherscan.io/address/0x84457ca9D0163FbC4bbfe4Dfbb20ba46e48DF254
+
+todo: this looks like an initialization but it wasn't present in the template
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000002` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** 
+
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x0000000000000000000000001126e5afb588d39c3c5465a15af389146d309581` <br/>
+  **After:** `0x000000000000000000000000ba2492e52f45651b60b8b38d4ea5e2390c64ffb1` <br/>
+  **Meaning:** Implementation address is set to the new `SystemConfig` per the [Superchain Registry][system-config-registry] and [Etherscan][system-config-etherscan].
+
+todo: this one has 12 events vs 5 events in the template, but it should be fine
+
+- **Key:** `0xa11ee3ab75b40e88a0105e935d17cd36c8faee0138320d776c411291bdbbb19f` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x000000000000000000000000000000000000000000000000000000000042b1d7` <br/>
+  **Meaning:** Sets `startBlock` at slot to 4370903. This should be the block number at which the `SystemConfig` proxy was initialized for the first time. [Etherscan events](https://sepolia.etherscan.io/address/0xf272670eb55e895584501d564AfEB048bEd26194#events) shows twelve events have been emitted since contract creation in block 4370901, and that the first `Initialize` event after that should be in block 4370903. Verification of the key can be done by ensuring the result of the [START_BLOCK_SLOT](https://sepolia.etherscan.io/address/0xba2492e52F45651B60B8B38d4Ea5E2390C64Ffb1#readContract#F8) getter on the implementation contract matches the key.
+
+
+### `0xfd0Bf71F60660E2f608ed56e1659C450eB113120` (`L1StandardBridgeProxy`)
+
+todo: simulated state changes 4 vs 5 in template
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xfd0Bf71F60660E2f608ed56e1659C450eB113120)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L5)
+
+State Changes:
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000003` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** The `initialized` boolean is set to `true`. The correctness of
```

Comment:
Prior value of 3 is again likely because of the `reinitializer` change

Reply:
Now the L1StandardBridge was [upgraded](https://github.com/base-org/contract-deployments/blob/26783846e57316b02ed511d75edf4bf92dcc4269/sepolia/2023-10-07-upgrade-l1-standard-bridge/.env) after initial deployment. Here's the [comparison](https://github.com/ethereum-optimism/optimism/compare/e3ba24e72085d85bb5584dda33a03ccf60db86f0..e6ef3a900c42c8722e72c2e2314027f85d12ced5) from that commit and MCP.

Reply:
changes look safe